docs/user-lifecycle.md

```diff
@@ -0,0 +1,63 @@
+# User lifecycle rules
+
+This document outlines the rules for managing user accounts within the TechPass system. These rules apply to all user accounts unless specific conditions are met that overwrite the general rules.
+
+## General rules
+
+### When a new user is created or invited
+
+1. **No sign-in within 23 days from creation date**:
+   - We send daily reminders to the user that their account will be terminated.
+   - If the user does not sign in by **day 30**, we stop reminders and terminate the user account.
+   - Five days after termination, the user account is deleted.
+
+### If the user has signed in before
+
+1. **No new sign-in in the past 60 days**:
+   - We send daily reminders to the user that their account will be disabled.
+   - If the user does not sign in by **day 90**, we stop reminders and disable the user account.
+
+2. **No new sign-in in the past 173 days**:
+   - We send daily reminders to the user that their account will be terminated.
+   - If the user does not sign in by **day 180**, we stop reminders and terminate the user account.
+   - Five days after termination, the user account is deleted.
```

Comment:
I would remove this line to avoid confusing it further with the soft delete feature in AAD.